spec/indexers/hyrax/file_set_indexer_spec.rb

```diff
@@ -2,148 +2,249 @@
 RSpec.describe Hyrax::FileSetIndexer do
   include Hyrax::FactoryHelpers
 
+  let(:fileset_id) { 'fs1' }
```

Comment:
This spec used to test the indexer for the AF `::FileSet`, but now tests the indexer for the Valkyrie `Hyrax::FileSet`.